client/rest-high-level/src/test/java/org/elasticsearch/client/ClusterClientIT.java

```diff
@@ -136,4 +141,38 @@ public void testListTasks() throws IOException {
         }
         assertTrue("List tasks were not found", listTasksFound);
     }
+
+    public void testPutPipeline() throws IOException {
+        String id = "some_pipeline_id";
+        XContentType xContentType = randomFrom(XContentType.values());
+        XContentBuilder pipelineBuilder = XContentBuilder.builder(xContentType.xContent());
+        pipelineBuilder.startObject().field(Pipeline.DESCRIPTION_KEY, "some random set of processors");
+        pipelineBuilder.startArray(Pipeline.PROCESSORS_KEY);
+        //Start first processor
+        pipelineBuilder.startObject();
```

Comment:
I tend to like doing something like this:
```
builder.startObject();
{
    builder.field(Pipeline.DESCRIPTION_KEY, "some random set of processors");
    builder.startArray(Pipeline.PROCESSORS_KEY);
    {
        pipelineBuilder.startObject().startObject("set").field("field", "foo").field("value", "bar").endObject().endObject();
        pipelineBuilder.startObject().startObject("convert").field("field", "rank").field("type", "integer").endObject().endObject();
    }
}
```

The `{` and `}` makes block which beg for the "normal" indentation you'd get in json so it kind of lines up.

Reply:
Thanks for the review. I only saw one thing to do (you mentioned two). Made the change for that.